chia/types/blockchain_format/program.py

```diff
@@ -243,13 +241,13 @@ def _sexp_replace(
         if len(kwargs) > 1:
             raise ValueError("conflicting paths")
         # yes, the terminal case is hinted incorrectly for now
```

Comment:
this comment to delete also?

Reply:
Are these properly hinted?  `kwargs` is hinted as `Any` not `Program` so there's nothing saying that this return path is in fact going to match the function return hint.  I could re-explore why we simplified this to `T_Program` instead of the more complicated type in the comment above the `def`, but I don't recall offhand.

Reply:
```suggestion
```